src/objects/aff/zcl_abapgit_object_common_aff.clas.abap

```diff
@@ -612,6 +623,24 @@ CLASS zcl_abapgit_object_common_aff IMPLEMENTATION.
       CATCH cx_root INTO lx_exception.
         zcx_abapgit_exception=>raise_with_text( lx_exception ).
     ENDTRY.
+  ENDMETHOD.
+
+  METHOD remove_abap_language_version.
+    DATA lv_pattern TYPE string.
+    DATA lv_string TYPE string.
+
+    lv_string = zcl_abapgit_convert=>xstring_to_string_utf8( iv_data = iv_json_as_xstring ).
 
+    lv_pattern = ',\s*"abapLanguageVersion"\s*:\s*"[^"]+"'.
+    " Replace the first occurence of "abapLanguageVersion" (in the header) only, so that
+    " further fields in the content can be still have the name "abapLanguageVersion"
```

Comment:
This assumes a certain order but JSON generally does not guarantee that. I'm not sure why there should be other alav fields since we don't want to serialize them anyway

Reply:
Using AJSON parser, now.